readthedocs/tutorial.rst

```diff
@@ -240,15 +257,13 @@ The same server also offers a secure FTP (``sftp``) service for testing. We can
 .. code-block:: bash
 
       # list the files on the sftp server
-      xenon filesystem sftp --location test.rebex.net --username demo --password password list --long /
+      xenon filesystem sftp --location test.rebex.net --username demo --password password list /
 
       # download a file from the sftp server
-      xenon filesystem sftp --location test.rebex.net --username demo --password password download --replace /readme.txt `pwd`/readme.txt 
+      xenon filesystem sftp --location test.rebex.net --username demo --password password download /readme.txt `pwd`/readme2.txt 
```

Comment:
``pwd`` @jmaassen this is a break from the earlier pattern...generally not a good thing from education point of view

Reply:
Agreed. Will fix.